src/textual/screen.py

```diff
@@ -1067,3 +1067,15 @@ def __init__(
     ) -> None:
         super().__init__(name=name, id=id, classes=classes)
         self._modal = True
+
+
+class _SystemModalScreen(ModalScreen[ScreenResultType], inherit_css=False):
```

Comment:
Shouldn't this be more of a mixin kind of thing, `_SystemScreen` or something like that?

It'd let `CommandPalette` inherit cleanly from `ModalScreen` and it'd let the coming system screens inherit from whatever type of screen they want (regular one or modal).

Reply:
> Shouldn't this be more of a mixin

I'll let *you* have that conversation with Will. :-)